backend/FwLite/LocalWebApp/SyncService.cs

```diff
@@ -11,15 +12,32 @@ namespace LocalWebApp;
 public class SyncService(
     DataModel dataModel,
     CrdtHttpSyncService remoteSyncServiceServer,
-    AuthHelpersFactory factory,
+    AuthHelpersFactory authHelpersFactory,
     CurrentProjectService currentProjectService,
     ChangeEventBus changeEventBus,
     IMiniLcmApi lexboxApi,
     ILogger<SyncService> logger)
 {
     public async Task<SyncResults> ExecuteSync()
     {
-        var remoteModel = await remoteSyncServiceServer.CreateProjectSyncable(await currentProjectService.GetProjectData());
+        var project = await currentProjectService.GetProjectData();
+        if (string.IsNullOrEmpty(project.OriginDomain))
+        {
+            logger.LogWarning("Project {ProjectName} has no origin domain, unable to create http sync client",
+                project.Name);
+            return new SyncResults([], [], false);
```

Comment:
Maybe introduce a `static readonly NOOP_RESULT = new SyncResults([], [], false)`? Up to you.
In theory callers could then check if they got a `SyncService.NOOP_RESULT`.

Reply:
I like the idea, though I was actually wanting to add a reason to why it didn't sync so that if the receiver was going to throw an error due to not syncing then it could have a reason why. Right now you just need to look at the logs to see the warning logged before.

Reply:
that change will have to be made in the harmony repo though :(

Reply:
Sounds worth doing.